website/docs/advanced-concepts/user-provisioning-group-sync.mdx

```diff
@@ -0,0 +1,109 @@
+---
+description: Guide on how to configure SCIM
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Setup SCIM Provisioning
+
+:::info
+SCIM Provisioning is only available in Appsmith's [Enterprise Edition](https://www.appsmith.com/pricing).
+:::
+
+
+This page shows how to set up SCIM on your Appsmith instance to streamline the user lifecycle management based on your SSO provider.
+
+
+## Prerequisites
+
+- A self-hosted Appsmith instance. See the [installation guides](/getting-started/setup/installation-guides) for deploying Appsmith.
+- A SAML SSO provider configured on your Appsmith instance.
+
+## Enable SCIM provisioning on Appsmith
+
+1. On your instance, go to **Admin Settings > Access Control** and click **Provisioning**.
+2. On the **User Provisioning & Group Sync** page, click **Configure**.
+3. Copy the **SCIM API endpoint** URL to add it later in your IdP application.
+4. Under **API key to setup SCIM**, click the **Generate API key** button and copy the API key to add it later in your IdP application.
+
+
+## Configure SSO provider
+
+To complete the setup, go to your IdP application that is connected to Appsmith and follow the steps below:
+
+<Tabs queryString="scim">
+<TabItem label="Active Direcotry (SAML)" value="scim-ad-saml">
+
+1. On the Active Directory homepage, click **Enterprise application** on the sidebar and open the SAML application that is already connected to Appsmith.
+
+2. On the application **Overview** page, click **Provisioning** on the sidebar.
+
+3. On the **Provisioning** homepage, click **Provisioning** on the sidebar and follow the steps below:
+
+    a. Select **Automatic** from the **Provision Mode** dropdown.
+
+    b. In the **Tenant URL** field, paste the **SCIM API endpoint** URL that you copied in step 3 under [Enable SCIM provisioning](/advanced-concepts/user-provisioning-group-sync#enable-scim-provisioning-on-appsmith) section.
+
+    c. In the **Secret Token** field, paste the **API key** that you copied in step 4 under [Enable SCIM provisioning](/advanced-concepts/user-provisioning-group-sync#enable-scim-provisioning-on-appsmith) section.
+
+    d. Click the **Test Connection** button to check if the added credentials are authorized to enable provisioning.
+
```

Comment:
This section needs to have a separate heading

**Assign users and groups to the application**